docs/CostCenters.md

```diff
@@ -0,0 +1,102 @@
+<a name="CostCenters"></a>
+
+## CostCenters
+
+**Kind**: global class
+
+* [CostCenters](#CostCenters)
+  * [new CostCenters(sdk, request, baseUrl)](#new_CostCenters_new)
+  * [.addFacility(costCenterId, facilityId)](#CostCenters+addFacility) ⇒ <code>Promise</code>
+  * [.create(costCenter)](#CostCenters+create) ⇒ <code>Promise</code>
+  * [.getAll()](#CostCenters+getAll) ⇒ <code>Promise</code>
+
+<a name="new_CostCenters_new"></a>
+
+### new CostCenters(sdk, request, baseUrl)
+
+| Param   | Type                | Description                                                             |
+| ------- | ------------------- | ----------------------------------------------------------------------- |
+| sdk     | <code>Object</code> | An instance of the SDK so the module can communicate with other modules |
+| request | <code>Object</code> | An instance of the request module tied to this module's audience.       |
+| baseUrl | <code>string</code> | The base URL provided by the parent module                              |
+
+<a name="CostCenters+addFacility"></a>
+
+### costCenters.addFacility(costCenterId, facilityId) ⇒ <code>Promise</code>
+
+Adds a facility to a cost center
+
+API Endpoint: '/costcenters/:costCenterId/facilities/:facilityId'
+Method: POST
+
+**Kind**: instance method of [<code>CostCenters</code>](#CostCenters)  
+**Fulfill**: [<code>CostCenterFacility</code>](./Typedefs.md#CostCenterFacility) Information about the new cost center facility relationship  
+**Reject**: <code>Error</code>
+
+| Param        | Type                | Description                                 |
+| ------------ | ------------------- | ------------------------------------------- |
+| costCenterId | <code>string</code> | UUID corresponding with a facility grouping |
+| facilityId   | <code>number</code> | The ID of a facility                        |
+
+**Example**
+
+```js
+contxtSdk.facilities.costCenters
+  .addFacility('b3dbaae3-25dd-475b-80dc-66296630a8d0', 4)
+  .then((grouping) => console.log(grouping))
+  .catch((err) => console.log(err));
+```
+
+<a name="CostCenters+create"></a>
+
+### costCenters.create(costCenter) ⇒ <code>Promise</code>
+
+Creates a new cost center
+
+API Endpoint: '/costcenters'
+Method: POST
+
+**Kind**: instance method of [<code>CostCenters</code>](#CostCenters)  
+**Fulfill**: [<code>CostCenter</code>](./Typedefs.md#CostCenter) Information about the new cost center  
+**Reject**: <code>Error</code>
+
+| Param                     | Type                | Description |
+| ------------------------- | ------------------- | ----------- |
+| costCenter                | <code>Object</code> |             |
+| costCenter.description    | <code>string</code> |             |
+| costCenter.name           | <code>string</code> |             |
```

Comment:
Should name and description have a description of `string`? I'm not sure what we usually do here

Reply:
Looking at groupings, it is consistent.